tests/test_benchmark.py

```
@@ -0,0 +1,171 @@
+    ----------
+    num_nodes : int
+        Total number of tasks to generate.
+    seed : int
+        Random seed for reproducibility.
+
+    Returns
+    -------
+    list[Task]
+        List of Task objects in topological order.
+    """
+    rng = random.Random(seed)
```

Ruff (0.14.5)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)